ideas/142-recovery-compatibility/README.md

```diff
@@ -0,0 +1,75 @@
+## Preamble
+
+    Idea: #142
+    Title: Wallet Recovery Compatibility
+    Status: Draft
+    Created: 2018-04-04
+    Requires (*optional): #94 Wallet Compatibility
+    Replaces (*optional): <Idea number(s)>
+
+## Summary
+
+A user should be able to recover accounts created in MyCrypto, MetaMask, Toshi, Cipher Browser, and other
+mnemonic wallets.
+
+## Swarm Participants
+
+- Lead Contributor: @pilu
+- Testing & Evaluation: <!-- @username -->
+- Contributor: <!-- @username -->
+- Contributor: <!-- @username -->
+- PM: <!--- @username -->
+- UX (if relevant): <!-- @username -->
+<!-- - Contributor: @username -->
+
+## Product Overview
+
+The wallets named above create the private key with the mnemonic words but without password.
+
+In those wallets, the password is only used to lock the account, and it's not needed to recover it.
```

Comment:
Also to encrypt the private key, right? 

From MyEtherWallet:
```
This password encrypts your private key. This does not act as a seed to generate your keys. You will need this password + your private key to unlock your wallet.
```

Reply:
Exactly @adambabik I'll update it. I used that words because we lock the account by encrypting/decrypting the key file.